rocketpool-cli/commands/pdao/status.go

```diff
@@ -12,13 +12,15 @@ import (
 	"github.com/rocket-pool/rocketpool-go/v2/types"
 	"github.com/rocket-pool/smartnode/v2/rocketpool-cli/client"
 	"github.com/rocket-pool/smartnode/v2/rocketpool-cli/utils"
+	"github.com/rocket-pool/smartnode/v2/rocketpool-cli/utils/terminal"
 	"github.com/rocket-pool/smartnode/v2/shared/types/api"
 )
 
 const (
-	colorBlue             string = "\033[36m"
-	colorReset            string = "\033[0m"
-	colorGreen            string = "\033[32m"
+	colorBlue  string = terminal.ColorBlue
+	colorReset string = terminal.ColorReset
+	colorGreen string = terminal.ColorGreen
+
```

Comment:
heh. I meant get rid of these consts. Change references to them to refer to the consts in `terminal`.